src/lib/apiTypesV2/MqttInfo.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+*
+* Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Burak Karaboga
+*/
+#include <string>
+#include <map>
+
+#include "mongo/client/dbclient.h"
+#include "logMsg/logMsg.h"
+
+#include "common/JsonHelper.h"
+#include "mongoBackend/dbConstants.h"
+#include "mongoBackend/safeMongo.h"
+#include "apiTypesV2/MqttInfo.h"
+
+using namespace mongo;
```

Comment:
This construct goes against the style guide for Orion, I'm afraid.
Check out this doc: https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md

The idea behind this rule is that `mongo::XXX` is clearer than just `XXX`, showing clearly where `XXX` comes from.

Reply:
Correct. Check rules M12 and M12bis in the documented cited by @kzangeli 

What is legal according to style rule is to use `using` which the specific function you want to use, e.g.:

```
using mongo::BSONObj;
using mongo::BSONElement;
```

Take it into account in the case it could help.